Makefile

```diff
@@ -42,5 +46,8 @@ lint:
 
 image:
 		docker buildx create --use
-		docker buildx build --platform=linux/amd64,linux/arm64 --pull ${DOCKER_ARGS} .
-		docker buildx build --platform=linux/amd64 --load --tag '${NAME}:${DOCKER_VERSION}' .
+		docker buildx build --push --platform=linux/amd64,linux/arm64 ${DOCKER_ARGS} --tag '${DOCKER_NAME}:${DOCKER_VERSION}' .
```

Comment:
Do not create a temporary local image with short name `ims-kiwi-ng-opensuse-x86_64-builder`. instead, create an image with full name `artifactory.algol60.net/csm-docker/unstable/ims-kiwi-ng-opensuse-x86_64-builder` from the beginning.